### DISCUSSION TITLE
ArcGIS Pro 3 Compatibility

### DISCUSSION THREAD

**Discussion by imgannon** [usgs/gems-tools-pro#39] [General]
Hello, 

Is there a time frame when this toolbox will be tested for ArcGIS Pro 3.0?

I am in a situation where I am unable to open my arc project on anything less that version 3.0 since I made edits using that version. This means that I am also unable to utilize the GeMS toolbox to check topology and validate database.

**Comment by ethoms-usgs** [comment]
Can you tell me more about what is going on? I have been working on the toolbox since moving to Pro 3.0 a few months ago so it is already at 3. And I think any .tbx file should open in earlier versions. It's the code behind the parameter forms that might be an issue. Arcpy tools sometimes change slightly from version to version, but usually still work at older versions.

**Comment by imgannon** [comment]
Hi Evan,

Thank you for getting back to me so quickly.

My GeMS database was created using ArcGIS Pro 3.0 and my main issue is
that, so far, none of the tools in the GeMS toolbox seem to be working.
When I run the topology check and database validation tools, they fail and
produce error messages. I can send you the error messages if that helps. I
have downloaded the newest version of the toolbox today, so I do not think
that is the issue.

Thanks,
Ian

On Thu, Jan 19, 2023 at 12:41 PM Evan Thoms ***@***.***>
wrote:

> Can you tell me more about what is going on? I have been working on the
> toolbox since moving to Pro 3.0 a few months ago so it is already at 3. And
> I think any .tbx file should open in earlier versions. It's the code behind
> the parameter forms that might be an issue. Arcpy tools sometimes change
> slightly from version to version, but usually still work at older versions.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DOI-USGS/gems-tools-pro/discussions/39#discussioncomment-4730178>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5LET3OKYRP3AVO5IFZRCEDWTF4ENANCNFSM6AAAAAAUARJOQ4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Ian Gannon

Geological Engineering/Geology

P: (248) 736-2922

**Reply by ethoms-usgs** [reply]
Yes, please send me the error messages, or start an issue and post them there.

**Comment by imgannon** [comment]
This is the error message that I am receiving when running the validate
database tool.
[image: image.png]
And this is what I get for the check topology tool.
[image: image.png]

On Thu, Jan 19, 2023 at 6:01 PM Evan Thoms ***@***.***> wrote:

> Yes, please send me the error messages, or start an issue and post them
> there.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DOI-USGS/gems-tools-pro/discussions/39#discussioncomment-4732548>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5LET3PFFF6ZN3MYNPOOW5LWTHBUBANCNFSM6AAAAAAUARJOQ4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Ian Gannon

Geological Engineering/Geology

P: (248) 736-2922

**Reply by ethoms-usgs** [reply]
Shoot, the screenshots didn't upload. Can you copy the text or try sending the images to ethoms@usgs.gov?
